docs/endpoints/multi_user.rst

```diff
@@ -427,7 +427,20 @@ the Globus Connect Server's `Identity Mapping documentation`_.
 This file is the template that will be interpolated with user-specific variables for
 successful start-UEP requests.  More than simple interpolation, the MEP treats this file
 as a `Jinja template`_, so there is a good bit of flexibility available to the motivated
-administrator.  The initial user config template implements two user-specifiable
+administrator.
+
+.. tip::
+
+   Administrators can combine multiple templates with the ``extends``, ``include``, and
+   ``import`` Jinja tags.  However, since these templates are rendered in user space for
+   security reasons, the administrator must:
```

Comment:
"for security reasons" makes more sense here, not in the changelog.  Might make that a parenthetical, however, as it's not directly relevant.  What's relevant for this discussion is simply that they are rendered in user space.